src/main/kotlin/com/devfloor/untitled/comment/application/CommentService.kt

```diff
@@ -52,4 +53,9 @@ class CommentService(
     fun modifyByCommentId(commentId: Long, request: CommentModifyRequest) {
         showByCommentId(commentId).modifyContent(request.content)
     }
+
+    fun destroyByCommentId(commentId: Long) {
+        commentRepository.deleteById(commentId)
+        commentFavoriteRepository.deleteByComment(showByCommentId(commentId))
+    }
```

Comment:
이거 아마 외래키 참조 때문에 삭제 순서를 바꿔야 하는걸로 알고 있어!!
그리고 이건 개인적인 의견인데 괄호 안에 계속 메서드나 함수가 들어가면 가독성이 조금 어려운 것 같아서 체이닝으로 처리하는건 어떨까?

```suggestion
    fun destroyByCommentId(commentId: Long) {
        showByCommentId(commentId).run {
            commentFavoriteRepository.deleteByComment(it)
        }
        commentRepository.deleteById(commentId)
    }
```